diff --git a/exoplayer-amzn-2.10.6/library/sabr/src/main/java/com/google/android/exoplayer2/source/sabr/manifest/SabrManifest.java b/exoplayer-amzn-2.10.6/library/sabr/src/main/java/com/google/android/exoplayer2/source/sabr/manifest/SabrManifest.java
index ca03764..8da63cb 100644
--- a/exoplayer-amzn-2.10.6/library/sabr/src/main/java/com/google/android/exoplayer2/source/sabr/manifest/SabrManifest.java
+++ b/exoplayer-amzn-2.10.6/library/sabr/src/main/java/com/google/android/exoplayer2/source/sabr/manifest/SabrManifest.java
@@ -492,7 +492,7 @@ public class SabrManifest implements FilterableManifest<SabrManifest> {
             if (result.length() > 1) result.append(',');
             result.append(range.hasFormatId() ? formatKey(range.getFormatId()) : "-")
                     .append(':')
-                    .append(range.hasDurationMs() ? range.getDurationMs() : -1);
+                    .append(range.getDurationMs());
         }
         return result.append(']').toString();
     }
